Looms.PoS/Looms.PoS.Application/Utilities/BusinessHours.cs

```diff
@@ -0,0 +1,6 @@
+namespace Looms.PoS.Application.Utilities;
+public class BusinessHours
+{
+    public int StartHour { get; set; } = 9;
+    public int EndHour { get; set; } = 17;
+}
```

Comment:
Should add it to business dao as each business working hours differ.

Reply:
added business hours to business dao